db/migrate/20181029020908_create_users.rb

```diff
@@ -0,0 +1,10 @@
+class CreateUsers < ActiveRecord::Migration[5.2]
+  def change
+    create_table :users do |t|
+      t.string :name
+      t.string :email
+
```

Comment:
empty line here

Reply:
this is also auto-generated and in other files there are also empty lines before timestamps